bodhi/server/migrations/versions/d3f8bd499ecd_add_from_tag_column_to_updates.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+Add from_tag column to updates.
+
+Revision ID: d3f8bd499ecd
+Revises: e8a059156d38
+Create Date: 2019-05-14 15:10:54.769789
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd3f8bd499ecd'
+down_revision = 'e8a059156d38'
+
+
+def upgrade():
+    """Add the from_tag column to the updates table."""
+    op.add_column('updates', sa.Column('from_tag', sa.UnicodeText(), nullable=True))
```

Comment:
I'm a very "from the bottom up" person I guess and called it `from_tag` and not `from_sidetag` because the code doesn't care if the tag you supply is a side tag or not—and AIUI can't distinguish between them either: it's just a matter of convention that we call some tags "side tags" because they're created ad-hoc and short-lived for the purpose of building packages "on the side". If you want the purpose of this API be better or explicitly reflected in the name of the flag, I'll happily make the change.

Reply:
I don't feel strongly on that one to be honest, so let's keep it as it is :+1: